.circleci/config.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+
+references:
+  environment: &environment
+    IMAGE_NAME: "zappi/calico-route-reflector-init"
+
+jobs:
+  test:
+    docker:
+      - image: docker:18.06.3-ce-git
+    environment: *environment
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+          version: 18.06.0-ce
+      - run:
+          name: "Build Image"
+          command: docker build -t $IMAGE_NAME .
+      - run:
+          name: "Test Image"
+          command: docker run --entrypoint "/bin/ls" $IMAGE_NAME init.sh
```

Comment:
Won't this fail? 🤔 Because none of these are set ...

```bash
: ${KUBERNETES_NODE_NAME:?"You need to set the KUBERNETES_NODE_NAME environment variable."}
: ${CALICO_ROUTE_REFLECTOR_CLUSTER_ID:?"You need to set the CALICO_ROUTE_REFLECTOR_CLUSTER_ID environment variable."}
: ${CALICO_ROUTE_REFLECTOR_LABEL:?"You need to set the CALICO_ROUTE_REFLECTOR_LABEL environment variable."}
```

Reply:
Nope, because here I'm just performing `ls` on `init.sh` to check for the existence of the script. It's not the best test, but it's okay for now.

Reply:
Right, I didn't see the `"/bin/ls"` entry point.